src/content/en/updates/2017/10/remove-shadow-piercing.md

```diff
@@ -0,0 +1,60 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Say goodbye to shadow-piercing CSS selectors.
+
+{# wf_updated_on: 2017-10-24 #}
+{# wf_published_on: 2017-10-24 #}
+{# wf_tags: webcomponents,shadowdom,style,css,deprecations,removals #}
+{# wf_featured_image: /web/updates/images/generic/styles.png #}
+{# wf_featured_snippet: Say goodbye to shadow-piercing CSS selectors. #}
+{# wf_blink_components: Blink>DOM #}
+
+# Removing ::shadow and /deep/ in Chrome 63 {: .page-title }
+
+{% include "web/_shared/contributors/robdodson.html" %}
+
+Starting in Chrome 63, the shadow-piercing selectors `::shadow` and `/deep/`
+will no longer be able to style content inside of a shadow root.
+
+- The `/deep/` combinator will act as a descendant selector. `x-foo /deep/ div`
+will work like `x-foo div`.
+- The `::shadow` pseudo-element will not match any elements.
+
+Note: If your site uses Polymer, the team has put together [a thorough guide](https://www.polymer-project.org/blog/2017-10-18-upcoming-changes.html)
```

Comment:
This link isn't working for me in the staging server. Might be just the nature of the staging server, but I'm also wondering if maybe notes have specific linking requirements?